llvm/lib/Target/AMDGPU/AMDGPUSubtarget.cpp

```diff
@@ -642,6 +642,17 @@ void GCNSubtarget::overrideSchedPolicy(MachineSchedPolicy &Policy,
     Policy.ShouldTrackLaneMasks = true;
 }
 
+void GCNSubtarget::mirFileLoaded(MachineFunction &MF) const {
```

Comment:
I think we should switch to using HwMode on the wavesize to swap out the actual operands on construction